solr_conf/solrconfig.xml

```diff
@@ -774,8 +775,9 @@
           subtitle^35
           author^100
           notes^50
-          content_nostem^50
+          content_nostem^100
           content
```

Comment:
Actually, I think we also should boost the regular content so that it will have priority over matches in the simple tokenized content. That probably means you need to boost the nostem content even more.